### PR TITLE
Enh/prototype webfrontend 5

### PR DESCRIPTION
**Pull request by mooxl**
This contains an early version of the GUI.

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE <!DOCTYPE html>
```
Comment (Paeti) [inline web/index.html]: ~~<!DOCTYPE~~

File: web/custom.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+};
+//Enables webcam
+navigator.mediaDevices.getUserMedia(constraints)
+  .then((stream) => {
+    webcam.srcObject = stream;
+  });
+
+// Checks if "^" Key is pressed
+document.addEventListener("keydown", function (e) {
+  if (e.keyCode == 192) {
```
Comment (Paeti) [inline web/custom.js]: Why keyCode = 192 (`)? Just random pick?

File: web/custom.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    // Draw the video frame to the canvas and make second stage visible.
+    context.drawImage(webcam, 0, 0, canvas.width, canvas.height);
+    user.style.opacity = 0;
+    user.style.transitionDuration = "1s";
+    loader.style.opacity = 1;
+    loader.style.transitionDuration = "1s";
+    smallerText.style.opacity = 0;
+    smallerText.style.transitionDuration = "1s";
+    crypticWow("Ihr Bild wird nun verarbeitet");
+  } else if (e.keyCode == 70) {
```
Comment (Paeti) [inline web/custom.js]: Same about keyCode = 70 ("f" I guess)?

File: web/style.css [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    width: 100vw;
+}
+
+#userForm img {
+    height: 100%;
+    display: block;
+    margin: auto;
+}
+
+#canvas {
```
Comment (Paeti) [inline web/style.css]: Unused?

File: web/custom.js (hunk 2 above)
Comment (mooxl) [reply web/custom.js]: Yes, the keyCode was a random Pick by me and will be shown in the documentation.

File: web/custom.js (hunk 3 above)
Comment (mooxl) [reply web/custom.js]: This function will be triggered when the REST API sends the data. For now i simulated is when pressing "f"(finished).

**Pull request merged**